azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/setup/SetupViewModel.kt

```diff
@@ -34,22 +36,28 @@ internal class SetupViewModel(
     }
 
     fun exitComposite() {
+        // double check here if we need both the action to execute
         dispatchAction(action = CallingAction.CallEndRequested())
         dispatchAction(action = NavigationAction.Exit())
     }
 
     override fun init(coroutineScope: CoroutineScope) {
         val state = store.getCurrentState()
+
+        if (state.localParticipantState.cameraState.operation == CameraOperationalStatus.ON) {
```

Comment:
It looks like a hack. Why do we have such case when CameraOperationalStatus.ON and preview is not on?

Reply:
In AppReduxState, we are initializing with localOptions value of camera & mic status. And implying camera state as needed in setup or call screen. 

If permission gets denied, camera & mic operation status gets updated with redux flow at permission manager.